api/Document.json

```
@@ -9756,40 +9756,40 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/indices.html#event-readystatechange",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "1"
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": "18"
             },
             "edge": {
               "version_added": "12"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "9"
```

I tried to confirm 9 with http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=9673 but it looks like it was first in Firefox 4:

```suggestion
              "version_added": "4"
```